doc/grpc-proxy.md

```diff
@@ -17,7 +17,7 @@
 #
 -->
 
-[中文](zh-cn/grpc-proxy.md)
+[Chinese](zh-cn/grpc-proxy.md)
```

Comment:
It would be better if you leave this unchanged.

Reply:
+1